pkgs/servers/sql/postgresql/ext/plv8.nix

```diff
@@ -11,26 +11,26 @@ stdenv.mkDerivation rec {
     owner = "plv8";
     repo = "plv8";
     rev = "v${version}";
-    sha256 = "0bv2b8xxdqqhj6nwyc8kwhi5m5i7i1yl078sk3bnnc84b0mnza5x";
+    sha256 = "1yi1ibiibvd0x4z5dm698w32ljrj3yr4j25jm1zkgkwd4ii8y644";
   };
 
-  makeFlags = [ "--makefile=Makefile.shared" ];
+  makefile = "Makefile.shared";
+
+  buildFlags = [ "all" ];
 
   preConfigure = ''
     patchShebangs ./generate_upgrade.sh
   '';
 
-  buildPhase = "make -f Makefile.shared all";
-
   installPhase = ''
     mkdir -p $out/bin
```

Comment:
should this be here? we don't put anything into it.

```
[nix-shell:~/.cache/nix-review/pr-64227]$ tree results/*
results/plv8
├── bin
├── lib
│   └── plv8-2.3.12.so
└── share
    └── postgresql
        └── extension
            ├── plcoffee--2.3.12.sql
            ├── plcoffee.control
            ├── plls--2.3.12.sql
            ├── plls.control
            ├── plv8--2.3.12.sql
            └── plv8.control

5 directories, 7 files
```

Reply:
yes, it's for buildEnv to behave correctly. See https://github.com/NixOS/nixpkgs/issues/22653

Reply:
Interesting, I would still put a comment as that's not immediately apparent. :)

Reply:
Sure, I added a comment